### ISSUE TITLE
jupyterlmod >1.9.0 does not work

### ISSUE THREAD

**Issue by dr-br** [cmd-ntrf/jupyter-lmod#25]
Jupyter Lmod > 1.9.0 does not work.
In Notebook, the "Softwares" tab remains empty. With jupyterlmod==2.0.0 browser says:
`Uncaught (in promise) SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data`

```
jupyter --version
jupyter core     : 4.6.3
jupyter-notebook : 6.1.4
qtconsole        : 4.7.7
ipython          : 7.16.1
ipykernel        : 5.3.4
jupyter client   : 6.1.7
jupyter lab      : 2.2.8
nbconvert        : 6.0.5
ipywidgets       : 7.5.1
nbformat         : 5.0.7
traitlets        : 4.3.3
```

```
pip freeze|grep jupyter
jupyter==1.0.0
jupyter-client==6.1.7
jupyter-console==6.2.0
jupyter-core==4.6.3
jupyter-server-proxy==1.5.0
jupyterlab==2.2.8
jupyterlab-pygments==0.1.1
jupyterlab-server==1.2.0
jupyterlmod==1.9.0 # WORKING
jupyterlmod==2.0.0 # NOT WORKING
```

```
module --version

Modules based on Lua: Version 8.3.15  2020-06-05 12:32 -05:00
    by Robert McLay mclay@tacc.utexas.edu
```
Any help/hint is appreciated.

**Comment by cmd-ntrf** [comment]
Hi, 

Sorry for the late reply, I have been away for the past few weeks.

The error you are reporting with jupyterlmod 2.0.0 indicates jupyterlmod backend does not return a valid JSON string.
You can view the backend JSON response by replacing "lab" in the address bar by
- "lmod" - this should display the list of currently loaded modules
- "lmod/modules" - this should display the list of available modules

If you could provide the output display for both links, it will me figure why the backend answer is invalid.

**Comment by dr-br** [comment]
Thanks for the reply, I can test it next week.

**Comment by cmd-ntrf** [comment]
@dr-br  Any update ? Or I can close this issue?

**Comment by dr-br** [comment]
Hi @cmd-ntrf, I promise to test it next week!

**Comment by cmd-ntrf** [comment]
Fixed by #32
